donkeycar/parts/telemetry.py

```diff
@@ -15,31 +14,40 @@
 from paho.mqtt.client import Client as MQTTClient
 
 
-class Telemetry(StreamHandler):
-    '''
+class MqttTelemetry(StreamHandler):
+    """
     Telemetry class collects telemetry from different parts of the system and periodically sends updated to the server.
     Telemetry reports are timestamped and stored in memory until it is pushed to the server
-    '''
+    """
 
-    def __init__(self, cfg, default_inputs = [], default_types = []):
+    def __init__(self, cfg, default_inputs=None, default_types=None):
```

Comment:
Why are you changing the defaults to `None` ? I think an empty `[]` might be a better default.

Reply:
Just to ignore the `default value is mutable` warning (https://florimond.dev/blog/articles/2018/08/python-mutable-defaults-are-the-source-of-all-evil/). If you insist - I can roll it back